GraphQL.md

```diff
@@ -19,47 +19,70 @@ POST https://yourdomain.com/graphql
 
-  * `op` (abbreviated for `operation`) is one of `FETCH`, `REPLACE`, `ADD`, or `DELETE`. The default value is `FETCH` when unspecified.
-  * `id` is the identifier for an object. If this value is unspecified the system assumes to be working on a collection of these objects. Otherwise, it is working on the specified id.
-  * `data` is the data input object that is used as the argument to the specified `op` argument. This argument is unused in the case of `FETCH` and `DELETE`.
+  * `op` (abbreviated for `operation`) is one of `UPSERT`, `REPLACE`, `REMOVE` or `DELETE`. The default value is `UPSERT` when unspecified.
+
+     * `UPSERT` - inserts new or edits existing members of a relationship. It is the primary mechanism by which any updates will be made to the model.
+     * `REPLACE` - can be thought as a combination of `UPSERT` and `REMOVE`, wherein, any objects not specified in the data argument (that are visible to the user) will be removed.
+     * `REMOVE` - only disassociates a relationship. If the underlying store automatically deletes orphans, it will still be deleted.
+     * `DELETE` - both disassociates the relationship and it deletes the entity from the persistence store.
+
+  
+  * `ids` are used as the ientifier for object(s). This field can hold a singleton value (for a single object) or a list of values to specify multiple ids. If this value is unspecified the system assumes to be working on a collection of these objects. Otherwise, it is working on the specified id.
+  *  `data` is the data input object that is used as the argument to the specified `op` argument. This argument is unsupported in the case of `REMOVE`, `DELETE` or while _fetching_ an existing object.
 
 Finally, the `exposedFields` is a specified list of values for the object(s) that the caller expects to be returned.
 
 # Examples
-Below are several examples. For each, assume a simple model of `Book` and `Author`. Particularly, the schema for each can be considered:
+Below are several examples. For each, assume a simple model of `Book`, `Author` and `Publisher`. Particularly, the schema for each can be considered:
 
 ```java
-@Include(rootLevel = true)
 @Entity
+@Table(name = "book")
+@Include(rootLevel = true)
 public class Book {
     @Id public long id;
     public String title;
+    @OneToMany
     public Set<Author> authors;
 }
 ```
-
 ```java
-@Include(rootLevel = false)
 @Entity
+@Table(name = "author")
+@Include(rootLevel = false)
 public class Author {
     @Id public long id;
     public String name;
+    @OneToMany
+    public Set<Book> books;
+    @OneToMany
+    public Set<Publisher> publishers;
+}
+```
+```java
+@Entity
+@Table(name = "publisher")
+@Include(rootLevel = false)
+public class Publisher {
+    @Id public long id;
+    public String name;
+    @OneToMany
     public Set<Book> books;
 }
 ```
 
-## Fetches
-
-A set of `FETCH` examples.
-
-### Collection of Books with ID, Title, and Authors
+## UPSERT
```

Comment:
This title belongs down below where ### Add is listed.  It should replace the Add title